references/contract-addresses.mdx

```diff
@@ -138,7 +138,7 @@ and all contracts on Ethereum Mainnet besides the above are paused.
 - BondingManager (Proxy):
   [0xe42229d764F673EB3FB8B9a56016C2a4DA45ffd7](https://testnet.arbiscan.io/address/0xe42229d764F673EB3FB8B9a56016C2a4DA45ffd7)
 - BondingManager (Target):
-  [0x53ebC58f6D5c326fa29c6ce58B16997E7A0b95D1](https://testnet.arbiscan.io/address/0x53ebC58f6D5c326fa29c6ce58B16997E7A0b95D1)
```

Comment:
Looks like this is updating an address under "Arbitrum Rinkeby", but should instead be updating the address under "Arbitrum Mainnet".

Reply:
lol good catch. both addresses started with 0x5 and I confused them 😅